lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2439,14 +2690,188 @@ mockProtocolParametersForBalancing = (mockProtocolParameters, nodePParams)
         , Cardano.protocolParamMonetaryExpansion = 0
         , Cardano.protocolParamTreasuryCut  = 0
         , Cardano.protocolParamUTxOCostPerWord = Just 34482
-        , Cardano.protocolParamCostModels = Map.empty -- TODO
+        , Cardano.protocolParamCostModels =
+            Map.singleton
+                (Cardano.AnyPlutusScriptVersion Cardano.PlutusScriptV1)
+                costModel
         , Cardano.protocolParamPrices =
-            Just $ Cardano.ExecutionUnitPrices 1 1
+            Just $ Cardano.ExecutionUnitPrices
+                (721 % 10000000)
+                (577 % 10000)
```

Comment:
@Anviking would it be possible to add a comment saying where these particular values come from?

Reply:
👍 Extended the comment on `mockProtocolParametersForBalancing`